src/DacpacTool/ModelValidationError.cs

```diff
@@ -72,5 +72,30 @@ public override string ToString()
 
             return stringBuilder.ToString();
         }
+
+        /// <summary>
+        /// Make message for writing to output during build process more similar to the one displayed by standard SSDT build
+        /// </summary>
+        /// <param name="severity">Severity to override depends on TreatTSqlWarningsAsErrors</param>
+        /// <returns></returns>
+        public string GetOutputMessage(ModelErrorSeverity severity)
```

Comment:
The original goal of the override of `ToString()` was to do exactly this. Perhaps that override is no longer necessary if we add this?

Reply:
IMHO override of `ToString` is good practice for develop & debug purpose.